src/python/sdk/examples/event.py

```diff
@@ -68,13 +69,14 @@ def get_channel() -> str:
 
 
 def start():
-    host_rest = get_rest_host()
-    channel = get_channel()
-    response = httpx.post(f"{host_rest}/rpc/jwt", json={"channels": [channel]})
-
+    token = jwt.encode(
```

Comment:
How about having a `get_jwt_secret` function like the ones above, with a hardcoded `JWT_SECRET_DEFAULT`?